sound/soc/sdca/sdca_asoc.c

```diff
@@ -786,6 +786,126 @@ int sdca_asoc_populate_dapm(struct device *dev, struct sdca_function_data *funct
 }
 EXPORT_SYMBOL_NS(sdca_asoc_populate_dapm, "SND_SOC_SDCA");
 
+static int sdca_soc_vol_reg_to_ctl(struct soc_mixer_control *mc, unsigned int reg_val,
```

Comment:
Maybe update the name to include q78 or something incase we need to as most custom handlers in the future.